packages/stores/src/ui-config/manage-liquidity/add-concentrated-liquidity.ts

```diff
@@ -275,20 +295,26 @@ export class ObservableAddConcentratedLiquidityConfig extends TxChainSetter {
     if (this.baseDepositOnly) return [new RatePretty(1), new RatePretty(0)];
     if (this.quoteDepositOnly) return [new RatePretty(0), new RatePretty(1)];
 
-    const quoteDeposit = new Int(1);
+    let amount1 = new Int(1000000);
```

Comment:
This appears to be assuming the amount has 6 decimals. Would that matter if say, amount1 is ETH with 18 decimals?

Reply:
If we want the 1 OSMO (with decimals, not uosmo), we could use something like `DecUtils.getTenExponentNInPrecisionRange(numDecimals)` which would yield `1000000` for OSMO.

Reply:
Asked in slack, but will also ask here: is there a frontend method that returns decimal amount given a denom?

Reply:
Yes those objects are of type "Currency" (or "AppCurrency") and here, you can get the currencies for the deposit assets like:
`this._baseDepositAmountIn.sendCurrency.coinDecimals`
`this._quoteDepositAmountIn.sendCurrency.coinDecimals`

Reply:
Addressed here, good catch https://github.com/osmosis-labs/osmosis-frontend/pull/1629/commits/ad1367242d115c31c7f38bb2e97c2174481e77e4